### DISCUSSION TITLE
How can I create chia plots with NFT in CLI

### DISCUSSION THREAD

**Discussion by leoliyuanduo** [Chia-Network/chia-blockchain#7193] [Q&A] [answered]
Trying to create a plot on a harvest machine. I plan to use same NFT created by my computer running node. Command was
'chia plots create -f farmKey-p poolKey -k 32 -t /temp/ -d /dist' If I update -c 'singleton address (pool contract address for plotting)' and remove 'p poolKey'. Does it work? Do I need to keep '-f farmKey'

**Comment by loppefaaret** [comment, marked as answer]
keep `-f`
replace `-p` with the "pay 2 singletin address" from `chia plotnft show` under `-c`

**Reply by leoliyuanduo** [reply]
Thanks

**Reply by heydari1** [reply]
Hi
would you please send me full command example for creating NFT plot on powershell, I tried with following :

.\chia.exe plotnft create -s pool -k 32 -b 5000 -u 128 -r 2 -t E:\temp2 -d D:\05KH117 -n 10

it didn't work

**Reply by loppefaaret** [reply]
first, you create one plot NFT - this plot NFT is created by a wallet, on the blockchain - you can then direct you new pool switchable plots at that plot NFT, and in turn, your wallet can create transactions to control where this plot NFT points to. any pool or yourself.
you then create plots pointing at the plot NFT, so whenever you switch the plot NFT, the plots that point to that plot NFT follows the switch
so assuming you create a plot NFT, as described in the [Pool Guide](https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide) with `.\chia plotnft create -s pool -u https://bar.examplepool.org` you can then do `.\chia plotnft show` which lists the `pay 2 singleton address:`
that pay 2 singleton address is now used instead of the pool public key `-p` as a `-c` during plotting, so you would do `.\chia plots create -k 32 -u 128 -b 3400 -r 4 -t X:\chiatemp -d Z:\chiaplots -f <farmer public key> -c <pay 2 singleton address>`

**Reply by heydari1** [reply]
at first thanks loppefaaret for your prompt answer
As I understand:
1- before doing any things I should create one plot by GUI wallet as described in the Pool Guide after it I can create other plot by power shell
2- I have no information about " pay 2 singleton address:" would you please tell me what is this and how can I find it
3- by using following command:

 .\chia plots create -k 32 -u 128 -b 3400 -r 4 -t X:\chiatemp -d Z:\chiaplots -f <farmer public key> -c <pay 2 singleton address>

can I create NFT plot?

**Reply by heydari1** [reply]
another thing  which command I should use for create NFT plot:
a) .\chia plots create
or 
b) \chia plotnft create 

thanks for your help

**Reply by loppefaaret** [reply]
no, first, you want to create a plot NFT!!! either from the GUI, or from the command line `chia plotnft create`
once you got a plot NFT, you list it with `chia plotnft show`
that will list the pay2singletonaddress that you need for `-c` in your `chia plots create` command

**Reply by heydari1** [reply]
thank you so much for your help
now I'm creating first nft plot after it I will check pay2singletonaddress, I don't want to disturbe you bro but is it possible to send a picture from " pay2singletonaddress"

**Reply by heydari1** [reply]
that was very useful, thanks for your help, it has been solved. 👍
